docs/allclasses-index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE HTML>
```

Comment:
Generated JavaDoc should not be stored in the repo (it can be easily generated from sources)